core/primitives/src/crypto/signature.rs

```diff
@@ -47,20 +47,6 @@ impl ToBytes for SecretKey {
     }
 }
 
-/// Allows using PublicKey in std containers and macros.
```

Comment:
Why do we remove this? Is it because this is not used?

Reply:
there is a clippy error: derived equality but custom hash

Reply:
Wait then we should implement `PartialEq` instead of removing this

Reply:
it's also unused